website-components-playground/src/pages/custom-form-modal-page.js

```diff
@@ -66,10 +66,7 @@ const CustomFormModalPageExample = () => (
                   formControls={
                     <>
                       <CustomFormModalPage.FormSecondaryButton
-                        onClick={() => {
-                          formikProps.resetForm();
-                          setIsOpen(false);
-                        }}
+                        onClick={formikProps.handleReset}
```

Comment:
Is this because we don't intentionally want the secondary button to close the modal in this _playground_? 🤔

Reply:
Ok. Just wanted to understand the change.

Thanks 👍